docker-rancher/server.json

```diff
@@ -265,63 +218,57 @@
       }
     },
     {
-      "apiVersion": "2015-06-15",
       "type": "Microsoft.Compute/virtualMachines",
-      "name": "[concat(parameters('rancherNodeName'))]",
+      "apiVersion": "2017-03-30",
+      "name": "[concat(parameters('rancherServerName'))]",
       "location": "[parameters('location')]",
-      "copy": {
-        "name": "vmLoopNode",
-        "count": "[variables('rancherCount')]"
-      },
       "dependsOn": [
-        "[concat('Microsoft.Storage/storageAccounts/', variables('storageAccountName'))]",
-        "[concat('Microsoft.Network/networkInterfaces/', parameters('rancherNodeName'), '-nic')]"
+        "[concat('Microsoft.Network/networkInterfaces/', parameters('rancherServerName'), variables('NICPrefix'))]"
       ],
       "properties": {
         "hardwareProfile": {
           "vmSize": "[variables('vmSize')]"
         },
         "osProfile": {
-          "computerName": "[concat(parameters('rancherNodeName'))]",
+          "computerName": "[concat(parameters('rancherServerName'))]",
           "adminUsername": "[parameters('adminUsername')]",
           "adminPassword": "[parameters('adminPassword')]"
         },
         "storageProfile": {
           "imageReference": {
-            "publisher": "[variables('imagePublisher')]",
-            "offer": "[variables('imageOffer')]",
-            "sku": "[variables('ubuntuOSVersion')]",
-            "version": "latest"
+            "publisher": "[variables('osImagePublisher')]",
+            "offer": "[variables('osImageOffer')]",
+            "sku": "[parameters('osImageSKU')]",
+            "version": "[variables('osImageVersion')]"
           },
           "osDisk": {
-            "name": "[concat(parameters('rancherNodeName'),'-osdisk')]",
-            "vhd": {
-              "uri": "[concat('http://',variables('storageAccountName'),'.blob.core.windows.net/',variables('vmStorageAccountContainerName'),'/',parameters('rancherNodeName'),'-osdisk','.vhd')]"
-            },
+            "name": "[concat(parameters('rancherServerName'),'_Os_Disk_1')]",
```

Comment:
can you update the other templates in the sample to use managed disks as well?

Reply:
sure